### PR TITLE
Add logging of asciidoc/-tor errors

### PR DESCRIPTION
**Pull request by choeppler**
Add logging of the errors which asciidoc and asciidoctor output to their
stderr stream when converting asciidoc documents. Note that
asciidoctor's exit code may be SUCCESS even if there are ERROR messages
in its stderr output (tested with Asciidoctor 0.1.4 and 1.5.5).
Therefore log the stderr output whenever it is non-empty.

See #2399

### CODE REVIEW

File: helpers/content.go [hunk 1]
```diff
@@ -546,9 +546,19 @@ func getAsciidocContent(content []byte) []byte {
...
 	cmd.Stdout = &out
-	if err := cmd.Run(); err != nil {
+	cmd.Stderr = &cmderr
+	err := cmd.Run()
+	// asciidoctor has exit code 0 even if there are errors in stderr
+	// -> log stderr output regardless of state of err
+	for _, item := range strings.Split(string(cmderr.Bytes()), "\n") {
+		item := strings.TrimSpace(item)
+		if item != "" {
+			jww.ERROR.Println(item)
```
Comment (bep) [inline helpers/content.go]: This is good -- but it would be useful if the page path was in the error print.

Comment (choeppler) [inline helpers/content.go]: Thanks -- and I agree. Haven't found a good way to pass the page path (see comment in #2399). Can the page path somehow be retrieved from the RenderingContext?

Comment (bep) [inline helpers/content.go]: Ah, OK -- I didn't see the entire function, you have no page here. Yes, we should add some more context to the `RenderingContext` that can used to give better feedback ... We have `DocumentID` (which is useless for logging) and we could add `DocumentName` where we can put the page's fileName.

Can you do that as part of this PR? (we can fix the rest of the logging later)

Comment (choeppler) [inline helpers/content.go]: Yes, I can do that. Putting `page.Path` in `DocumentName` we can get error messages like
```
ERROR: 2016/10/13 09:47:12 content.go:560: asciidoctor: ERROR: post/fail.ad: line 139: mismatched macro: endif::backen-docbook[], expected endif::backend-docbook[]
```
I'll add a second commit to this pull request with the changes for you to review. Ok?

Comment (bep) [inline helpers/content.go]: Yes, that looks perfect, thanks.

Comment (choeppler) [inline helpers/content.go]: Good. Done. See 441a2c90888b4f8e6463780fbd3e0caaa66c1f8b

**Pull request closed without merging**